board/opendingux/package/dingux-commander/dingux-commander.mk

```diff
@@ -0,0 +1,20 @@
+################################################################################
+#
+# DinguxCommander
+#
+################################################################################
+
+DINGUX_COMMANDER_VERSION = stable
+DINGUX_COMMANDER_SITE = $(call github,glebm,rs97-commander,$(DINGUX_COMMANDER_VERSION))
+DINGUX_COMMANDER_DEPENDENCIES = sdl sdl_image freetype
+
+DINGUX_COMMANDER_TARGET_PLATFORM = rg350
+DINGUX_COMMANDER_CONF_OPTS += -DTARGET_PLATFORM=$(DINGUX_COMMANDER_TARGET_PLATFORM)
+
+define DINGUX_COMMANDER_INSTALL_TARGET_CMDS
+	mkdir -p $(BINARIES_DIR)/opks
+	cd $(@D) && ./package-opk.sh $(DINGUX_COMMANDER_TARGET_PLATFORM) $(DINGUX_COMMANDER_BUILDDIR) \
+	  $(BINARIES_DIR)/opks/commander.opk
+endef
```

Comment:
That's strange, why do you create an OPK at all?

I think you should add some ```install()``` rules to your rs97-commander's CMakeFiles.txt, then you won't need any custom install macro.

Something I noticed in your repo, is that you have SDL_ttf and SDL_gfx built-in. There are buildroot packages for these, so you should be able to link them dynamically.

Reply:
Commander is intended to be a default app in the data partition that the user can delete/upgrade independently of the rest of the firmware.

RG350 buildroot builds several OPK packages like this that become part of the data partition.

SDL_ttf:
This version uses independent DPI scaling for fonts (to support RetroFW's non-square-pixel screens). This requires a patch to SDL_ttf. The patch is now upstream is not on any devices / in any released version yet.

SDL_gfx:
We use only one tiny file from SDL_gfx and it has to be the latest version. The easiest to link it statically.

`install` rules:
Not sure how to do this correctly in CMake. Contributions are welcome.